velox/type/Filter.cpp

```diff
@@ -280,6 +280,35 @@ bool BigintRange::testingEquals(const Filter& other) const {
       (upper_ == otherBigintRange->upper_);
 }
 
+bool BigintRange::testBloomFilter(
+    const AbstractBloomFilter& bloomFilter,
+    const velox::Type& type) const {
+  // Don't test bloom filter for a wider range
+  if ((upper_ - lower_) > kMaxBloomFilterChecks)
+    return true;
+
+  switch (type.kind()) {
+    case TypeKind::INTEGER:
```

Comment:
What about the 16 bit?

Reply:
Arrow bloom filter doesn't support 16 bit int. Do we want to use the same hashInt32 for 16 bit too(may be not backward compatible if other engines implement hashInt16) or add support for hashInt16?